ui/apps/platform/src/Containers/Clusters/DiscoveredClusters/DiscoveredCluster.tsx

```diff
@@ -43,6 +43,17 @@ export function getStatusText(status: DiscoveredClusterStatus) {
     return statusMap[status] ?? statusMap.STATUS_UNSPECIFIED;
 }
 
+const statusTipMap: Record<DiscoveredClusterStatus, string> = {
+    STATUS_SECURED: 'This cluster is already secured.',
+    STATUS_UNSECURED: 'This cluster has been discovered by a cloud source, but is not yet secured.',
+    STATUS_UNSPECIFIED:
+        'This cluster has has been discovered by a cloud source, but has undetermined status because metadata collected from secured clusters does not guarantee a unique match. Sensor requires access to the AWS EC2 instance tags to determine the cluster state.',
```

Comment:
So there are two primary situations in which unspecified status will occur:

1. EKS clusters not having EC2 instance tags.
2. Old sensor versions not providing enough metadata.

The AWS comment is only relevant to 1. I wonder how to communicate this properly without making it seem that in scenario 2, the AWS instance tags would help. Perhaps "On AWS Sensor requires access to the AWS EC2 instance tags to determine the cluster state."?

Reply:
Okay, as discussed with Stephan on chat:
1. Delete duplicated word in first sentence.
2. Delete second sentence and let docs explain the case analysis, which includes older versions of sensors.

Although input from docs team is always welcome, better to have sought that before now.